esky/bdist_esky/f_py2app.py

```diff
@@ -110,30 +110,55 @@ def copy_to_bootstrap_env(src,dst=None):
                 #  have no interest in examining right now.  Eventually this
                 #  guard will be more conservative.
                 pass
+        bsdir = dist.bootstrap_dir
         copy_to_bootstrap_env("Contents/Resources/include")
         if sys.version_info[:2] < (3, 3):
             copy_to_bootstrap_env("Contents/Resources/lib/"+pydir+"/config")
         else:
             copy_to_bootstrap_env("Contents/Resources/lib/"+pydir+"/config-%d.%dm"
                                    % sys.version_info[:2])
+            # copy across the zip file that we need to run the boostrap application
+            # from the inner package.
+            python_name = 'python%d%d' % sys.version_info[:2]
+            bs_lib_dir = os.path.join('Contents', 'Resources', 'lib')
+            zip_name = os.path.join(bs_lib_dir, '{}.zip'.format(python_name))
+            copy_to_bootstrap_env(zip_name)
+
+            # Then we can remove a lot of the unused files from the
+            # zip file as the boostrap executable needs very little
+            zfname = os.path.join(bsdir, zip_name)
+            with tempfile.TemporaryDirectory() as tdir:
+                esky.util.extract_zipfile(zfname, tdir)
+                # remove the original as this has too much
+                os.remove(zfname)
+                # this is the minimal set needed for the python.zip
+                member_list = ['_weakrefset.pyc', 'abc.pyc', 'codecs.pyc', 'io.pyc']
+                for enc in os.listdir(os.path.join(tdir, 'encodings')):
+                    member_list.append(os.path.join('encodings',enc))
+                esky.util.create_zipfile(tdir, zfname, members=member_list)
```

Comment:
I guess this part isn't really required, since we just go straight from the tdir to the bsdir with zfname.